bip-musig2.mediawiki

```diff
@@ -145,13 +148,17 @@ However, if ''PartialSigVerify'' succeeds for all partial signatures then ''Part
 
 ==== Tweaking ====
 
+The aggregate public key can be ''tweaked'', which modifies the key as defined in the [[#tweaking-definition|Tweaking Definition]] subsection.
+In order to apply a tweak, the KeyGen Context output by ''KeyAgg'' is provided to the ''ApplyTweak'' algorithm with the ''is_xonly_t'' argument set to false for ordinary tweaking and true for X-only tweaking.
+The resulting KeyGen Context can be used to apply another tweak with ''ApplyTweak'' or obtain the aggregate public key with ''GetPubkey''.
```

Comment:
Should this mention GetPubkey33 already? I don't think it's really necessary to mention it here but...

Reply:
Hm, here I wanted to only mention how to get the public key for Schnorr verification after tweaking.